group-coordinator/src/test/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorRuntimeMetricsTest.java

```diff
@@ -48,15 +54,38 @@ public void testMetricNames() {
             metrics.metricName("event-queue-size", METRICS_GROUP),
             metrics.metricName("partition-load-time-max", METRICS_GROUP),
             metrics.metricName("partition-load-time-avg", METRICS_GROUP),
-            metrics.metricName("thread-idle-ratio-avg", METRICS_GROUP)
+            metrics.metricName("thread-idle-ratio-avg", METRICS_GROUP),
+            metrics.metricName("event-queue-time-ms-max", METRICS_GROUP),
+            metrics.metricName("event-queue-time-ms-p50", METRICS_GROUP),
+            metrics.metricName("event-queue-time-ms-p95", METRICS_GROUP),
+            metrics.metricName("event-queue-time-ms-p99", METRICS_GROUP),
+            metrics.metricName("event-queue-time-ms-p999", METRICS_GROUP),
+            metrics.metricName("event-processing-time-ms-max", METRICS_GROUP),
+            metrics.metricName("event-processing-time-ms-p50", METRICS_GROUP),
+            metrics.metricName("event-processing-time-ms-p95", METRICS_GROUP),
+            metrics.metricName("event-processing-time-ms-p99", METRICS_GROUP),
+            metrics.metricName("event-processing-time-ms-p999", METRICS_GROUP),
+            metrics.metricName("event-purgatory-time-ms-max", METRICS_GROUP),
+            metrics.metricName("event-purgatory-time-ms-p50", METRICS_GROUP),
+            metrics.metricName("event-purgatory-time-ms-p95", METRICS_GROUP),
+            metrics.metricName("event-purgatory-time-ms-p99", METRICS_GROUP),
+            metrics.metricName("event-purgatory-time-ms-p999", METRICS_GROUP),
+            metrics.metricName("batch-flush-time-ms-max", METRICS_GROUP),
+            metrics.metricName("batch-flush-time-ms-p50", METRICS_GROUP),
+            metrics.metricName("batch-flush-time-ms-p95", METRICS_GROUP),
+            metrics.metricName("batch-flush-time-ms-p99", METRICS_GROUP),
+            metrics.metricName("batch-flush-time-ms-p999", METRICS_GROUP)
```

Comment:
Could you please add those metrics to the description and to the KIP?

Reply:
updated https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol#KIP848:TheNextGenerationoftheConsumerRebalanceProtocol-BrokerMetrics